packages/terra-clinical-no-data-view/src/NoDataView.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import IconNoData from 'terra-icon/lib/icon/IconGlasses';
```

Comment:
I guess if we copy the file and entries from old terra and store it within clinical, and horseshoe it somehow into the image framework style we could.